docs/advanced/node-local-storage.md

```diff
@@ -0,0 +1,356 @@
+      storageClassName: "local-storage"
+      resources:
+        requests:
+          storage: 1Gi
+```
+
+The PVC should be listed and the PV bound:
+
+```sh
+$ kubectl get pods
+NAME           READY   STATUS    RESTARTS   AGE
+local-test-0   1/1     Running   0          54s
+local-test-1   1/1     Running   0          48s
+$ kubectl get pvc
+NAME                     STATUS   VOLUME        CAPACITY   ACCESS MODES   STORAGECLASS    AGE
+local-vol-local-test-0   Bound    worker-1-pv   10Gi       RWO            local-storage   7s
+local-vol-local-test-1   Bound    worker-0-pv   10Gi       RWO            local-storage   1s
+$ kubectl get pv
+NAME          CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                            STORAGECLASS    REASON   AGE
+worker-0-pv   10Gi       RWO            Retain           Bound    default/local-vol-local-test-1   local-storage            97m
+worker-1-pv   10Gi       RWO            Retain           Bound    default/local-vol-local-test-0   local-storage            97m
+```
+
+You can also connect to the pod and write to the volume (is mounted on
+`/usr/test-pod`).
+
+If you delete a pod, as it is part of this stateful set, it will be recreated and
+continue to use the volume (you can connect to the pod and write, you will see
+it in the host volume too).
+
+What if you want to delete this application and use the PVs for a different
+application?
+
+Basically, as the retain policy is set to retain (just to be safe), you need to
+do some manual operations documented here:
+	https://kubernetes.io/docs/concepts/storage/persistent-volumes/#retain
+
+Note that using the delete policy this is different, so you may want to choose
+the policy that better fits your use case.
+
+### Automatic local persistent volume creation and handling
+
+The example manually created the `Kind: PersistentVolume` for each node. Also,
+the PV will continue to exist even if the PVC is deleted and, therefore, the
+lifecycle of the PV needs to be handled manually.
+
+Those limitation don't scale well with the number of nodes and applications,
+therefore to that end an external provisioner that will automatically create
+those objects when nodes are available and manage the lifecycle of the volume
+was created by sig-storage:
+
+	https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/
+
+### Limitations
+
+There are some limitations you might want to have in mind
+
+1. Lokomotive Kubernetes currently only exposes `/mnt/` to the kubelet, so only that
+directory can be used for node local storage. You can **manually** mount different
+devices on `/mnt/sda` or paths inside `/mnt` to your convenience.
+
+1. Using `setup_raid = "true"` will create a single RAID 0 array with all
+the spare disks and mount it in `/mnt/`. Therefore all the node storage will be
+available to only one PV, thus it will be available to only one PVC and one pod.
+This is the only automated setup supported in lokomotive at the time of this
```

Comment:
```suggestion
This is the only automated setup supported in lokomotive at the time of writing.
```

Reply:
Why change `at the time of this writing` by `at the time of writing`?

Reply:
https://english.stackexchange.com/questions/199381/is-at-the-time-of-writing-correct `at the time of writing` is the better version of this phrase as far as I know.

Reply:
Oh, thanks!